perun-core/pom.xml

```diff
@@ -350,6 +350,35 @@
 			<version>3.3</version>
 		</dependency>
 
+		<dependency>
```

Comment:
Hi, are these dependencies really necessary. It almost double the original size (adds 40 MB). Also glassfish define hundreds of overlaping classes. I'm unsure about them, since it is only a warning.

```
[WARNING] glassfish-embedded-web-3.1.2.jar, mail-1.4.2.jar define 267 overlappping classes: 
[WARNING]   - javax.mail.internet.ContentDisposition
[WARNING]   - com.sun.mail.pop3.Protocol
[WARNING]   - com.sun.mail.iap.Response
[WARNING]   - com.sun.mail.smtp.DigestMD5
[WARNING]   - javax.mail.internet.MimePart
[WARNING]   - com.sun.mail.imap.IMAPMessage
[WARNING]   - javax.mail.Message
[WARNING]   - com.sun.mail.imap.Rights
[WARNING]   - com.sun.mail.util.SocketFetcher
[WARNING]   - com.sun.mail.pop3.POP3Store
[WARNING]   - 257 more...
[WARNING] glassfish-embedded-web-3.1.2.jar, jackson-core-asl-1.7.4.jar define 108 overlappping classes: 
[WARNING]   - org.codehaus.jackson.util.DefaultPrettyPrinter
[WARNING]   - org.codehaus.jackson.impl.JsonWriteContext
[WARNING]   - org.codehaus.jackson.annotate.JsonSubTypes
[WARNING]   - org.codehaus.jackson.io.IOContext
[WARNING]   - org.codehaus.jackson.impl.JsonParserMinimalBase$1
[WARNING]   - org.codehaus.jackson.sym.NameN
[WARNING]   - org.codehaus.jackson.util.JsonParserDelegate
[WARNING]   - org.codehaus.jackson.util.MinimalPrettyPrinter
[WARNING]   - org.codehaus.jackson.util.DefaultPrettyPrinter$Lf2SpacesIndenter
[WARNING]   - org.codehaus.jackson.impl.ByteSourceBootstrapper
[WARNING]   - 98 more...
[WARNING] glassfish-embedded-web-3.1.2.jar, slf4j-log4j12-1.5.8.jar define 3 overlappping classes: 
[WARNING]   - org.slf4j.impl.StaticMarkerBinder
[WARNING]   - org.slf4j.impl.StaticLoggerBinder
[WARNING]   - org.slf4j.impl.StaticMDCBinder
....list go on.....
```